src/gs-shell.c

```diff
@@ -582,6 +588,43 @@ initial_overview_load_done (GsOverviewPage *overview_page, gpointer data)
 	g_signal_emit (shell, signals[SIGNAL_LOADED], 0);
 }
 
+static void
+on_overview_categories_refreshed (GsOverviewPage *page, gpointer user_data)
```

Comment:
Nitpick: Would it be better to call this `on_overview_categories_loaded()` so it matches the signal name? Especially since `refreshed` is a separate signal.